Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -25,8 +25,26 @@ configured explicitly in your :ref:`site configuration <sitehandling>`.
     :file:`config.yaml` site configuration file (located in
     :file:`config/sites/<yoursite>/config.yaml`).
 
-Enhancers and aspects are an important concept in TYPO3 and they are used to map
+Enhancers and aspects are an important concept in TYPO3 and they are based on patterns to map
```

Comment:
I think this is treating a rare edge case and would clutter up the section to much

Reply:
No, patterns are always necessary to map the url parameters to a speaking url from the slug field of a table.